### PR TITLE
polish(zjow): Polish and Revise Quickstart - Installation.

### PR DESCRIPTION
**Pull request by zjowowen**

### CODE REVIEW

File: source/01_quickstart/installation_zh.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+
+开发版本
+--------------
+
+如果您需要从 Github 源码安装最新的 DI-engine 开发版本，可以使用如下方法：
+
+.. code-block:: bash
+
+    git clone https://github.com/opendilab/DI-engine.git
```
Comment (PaParaZz1) [inline source/01_quickstart/installation_zh.rst:58]: why don't `cd DI-engine` before installation

File: source/01_quickstart/installation_zh.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    pip install ./DI-engine/
+
```
Comment (PaParaZz1) [inline source/01_quickstart/installation_zh.rst:60]: add a tip for `--user` option

File: source/01_quickstart/installation_zh.rst [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+.. code-block:: bash
+
+    # install atari and box-2d related packages
+    pip install DI-engine[common_env]
+    # install unittest(pytest) related packages
+    pip install DI-engine[test]
+    # enable numba acceleration
+    pip install DI-engine[fast]
+    #install multi extra packages
```
Comment (PaParaZz1) [inline source/01_quickstart/installation_zh.rst:74]: there should be a space

File: source/01_quickstart/installation_zh.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    pip install DI-engine[common_env,test,fast]
```
Comment (PaParaZz1) [inline source/01_quickstart/installation_zh.rst:75]: add a tip for `pip install ‘DI-engine[test]’`

File: source/01_quickstart/installation_zh.rst [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
```
Comment (PaParaZz1) [inline source/01_quickstart/installation_zh.rst:76]: add a part for installation check

File: source/01_quickstart/installation_zh.rst [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+使用 Docker 运行
```
Comment (PaParaZz1) [inline source/01_quickstart/installation_zh.rst:77]: add installation duration comment

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: source/01_quickstart/first_rl_program_zh.rst [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+    from ding.framework.middleware import OffPolicyLearner, StepCollector, interaction_evaluator, data_pusher, eps_greedy_handler, CkptSaver
+
+    with task.start(async_mode=False, ctx=OnlineRLContext()):
+        task.use(interaction_evaluator(cfg, policy.eval_mode, evaluator_env))  # 评估流程，放在第一个是为了获得随机模型的评分作为基准值
+        task.use(eps_greedy_handler(cfg))  # 衰减探索-利用的概率
+        task.use(StepCollector(cfg, policy.collect_mode, collector_env))  # 采集环境数据
+        task.use(data_pusher(cfg, buffer_))  # 将数据保存到 buffer
+        task.use(OffPolicyLearner(cfg, policy.learn_mode, buffer_))  # 训练模型
+        task.use(CkptSaver(cfg, policy, train_freq=100))  # 保存模型
+        task.run(max_step=100000)  # 在评估流程中，如果发现模型表现已经超过了收敛值，这里将提前结束
```
[the remark this answers is not in the document]
Comment (zjowowen) [reply source/01_quickstart/first_rl_program_zh.rst]: Fixed.

File: source/01_quickstart/installation_zh.rst [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+发布版本
+--------------
+
+您可以使用以下命令安装 DI-engine 的稳定发行版：
+
+.. code-block:: bash
+
+    # Install the latest pip
+    pip install
```
[the remark this answers is not in the document]
Comment (zjowowen) [reply source/01_quickstart/installation_zh.rst]: Fixed for multiple OS.

File: source/01_quickstart/installation_zh.rst (hunk 1 above)
Comment (zjowowen) [reply source/01_quickstart/installation_zh.rst:69]: I am just afraid user may neglect the dot. It's safter.

File: source/01_quickstart/installation_zh.rst (hunk 2 above)
Comment (zjowowen) [reply source/01_quickstart/installation_zh.rst:82]: Fixed.

File: source/01_quickstart/installation_zh.rst (hunk 3 above)
Comment (zjowowen) [reply source/01_quickstart/installation_zh.rst]: Fixed.

File: source/01_quickstart/installation_zh.rst (hunk 4 above)
Comment (zjowowen) [reply source/01_quickstart/installation_zh.rst:97]: Fixed.

File: source/01_quickstart/installation_zh.rst (hunk 5 above)
Comment (zjowowen) [reply source/01_quickstart/installation_zh.rst:98]: Fixed.

File: source/01_quickstart/installation_zh.rst (hunk 6 above)
Comment (zjowowen) [reply source/01_quickstart/installation_zh.rst:112]: Fixed.

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**